WordPress/WordPressTest/Dashboard/BlazeCampaignViewModelTests.swift

```diff
@@ -12,7 +12,7 @@ final class BlazeCampaignViewModelTests: XCTestCase {
         XCTAssertEqual(viewModel.imageURL, URL(string: "https://i0.wp.com/public-api.wordpress.com/wpcom/v2/wordads/dsp/api/v1/dsp/creatives/56259/image?w=600&zoom=2"))
 
         // Then stats are displayed
-        XCTAssertEqual(viewModel.impressions, "1,000")
+        XCTAssertEqual(viewModel.impressions, 1_000.abbreviatedString())
```

Comment:
Thanks for looking into this @dvdchr.

By calling `abbreviatedString()` here, it seems like we are testing more at the implementation level than the behavior level. That is, reading the test one doesn't learn what the expected impressions value, only that it should be whatever `abbreviatedString()` returns.

For reference, here's the production code, which, unsurprisingly, calls `abbreviatedString()`:

https://github.com/wordpress-mobile/WordPress-iOS/blob/697186e8f3555d8e7ca6942dc08679df8a915850/WordPress/Classes/ViewRelated/Blog/Blog%20Dashboard/Cards/Blaze/DashboardBlazeCampaignView.swift#L144

This is probably okay practically, because we are not likely are we to change this formatting...

Still, I think we would be better served by making the application language and region deterministic, as you suggested in the PR description.

Reply:
> Still, I think we would be better served by making the application language and region deterministic, as you suggested in the PR description.

Plus 1 on this. While this fixes tests for Indonesian, I tried running `DashboardStatsViewModelTests` with Arabic as the locale, and it still failed.